Dockerfile

```diff
@@ -16,11 +16,9 @@ RUN apt-get update && apt-get install -y \
     yarn=1.22.1-1 \
     build-essential
 
-RUN yarn add --dev typescript@3.9.7 \
-    @angular/cli@10.0.4 \
+RUN yarn global add @angular/cli@10.0.4 \
```

Comment:
Adding the various CLIs locally-only seems to just plain not work. Global it is.

`wasm-pack` and `typescript` on the other hand, we can get away with as devDependencies.